packages/sources/trueusd/src/endpoint/trueusd.ts

```diff
@@ -1,52 +1,50 @@
-import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { AdapterError, Requester, Validator } from '@chainlink/ea-bootstrap'
 import { ExecuteWithConfig, Config, InputParameters } from '@chainlink/types'
 
 export const NAME = 'trueusd'
 
 export const supportedEndpoints = ['trueusd']
 
-export const endpointResultPaths = {
-  trueusd: 'totalTrust',
-}
-
-const customError = (data: ResponseSchema) => !data.success
-
-export const description = 'https://core-api.real-time-attest.trustexplorer.io/trusttoken/TrueUSD'
+export const description = 'https://api.real-time-attest.trustexplorer.io/chainlink/TrueUSD'
 
 export const inputParameters: InputParameters = {
-  resultPath: {
+  field: {
```

Comment:
Though I like the name `field`, `resultPath` is kind of a reserved name for this if you want to tie a `resultPath` to an `endpoint`.

To do this you would export a new variable for the mapping between the two:
```
export const endpointResultPaths = {
  trueusd: 'totalTrust',
}
```

If you do this, when the endpoint `trueusd` is given, it will automatically fill the resultPath.

Reply:
Resolving based on private conversation